### PR TITLE
Cloudshare fixed

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/32128

### CODE REVIEW

File: Packs/CloudShare/Integrations/CloudShare/CloudShare.py [hunk 1]
```diff
@@ -1180,7 +1180,8 @@ def main() -> None:
             'cloudshare-suspend-class-environments': suspend_class_environments_command,
             'cloudshare-modify-class': modify_class_command,
             'cloudshare-get-students': get_students_command,
-            'cloudshare-get-student': delete_student_command,
+            'cloudshare-get-student': get_student_command,
```
Comment (ShahafBenYakir) [inline Packs/CloudShare/Integrations/CloudShare/CloudShare.py:1183]:
```suggestion
            'cloudshare-get-student': get_student_command,    # This is here for maintaining BC
```

File: Packs/CloudShare/ReleaseNotes/1_0_2.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### CloudShare (Beta)
+- Fixed an issue where the ***cloudshare-delete-student*** snd the ***cloudshare-get-student*** command was incorrectly handled in the code.
```
Comment (ShahafBenYakir) [inline Packs/CloudShare/ReleaseNotes/1_0_2.md:4]:
```suggestion
- Fixed an issue where the ***cloudshare-delete-student*** and the ***cloudshare-get-student*** commands were incorrectly handled in the code.
```

File: Packs/CloudShare/Integrations/CloudShare/CloudShare.py (hunk 1 above)
Comment (moishce) [reply Packs/CloudShare/Integrations/CloudShare/CloudShare.py:1183]: It's not just for bc
"cloudshare-get-student" command exists in the yml and has a function in the code

Comment (ShahafBenYakir) [reply Packs/CloudShare/Integrations/CloudShare/CloudShare.py:1183]: Oh so in the YML there is 2 command (studen**ts** and studen**t**) and in the py file only one right? Let's change the comment to explain that `# Both commands (get-student and get-students) use the same function `
While you are at it please remove the paloaltonetworks URL from the metadata file

File: Packs/CloudShare/ReleaseNotes/1_0_1.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
 
 #### Integrations
 ##### CloudShare (Beta)
-- Fixed an issue where the **cloudshare-delete-class-environemnts** command was incorrectly handled in the code.
+- Fixed an issue where the **cloudshare-delete-class-environments** command was incorrectly handled in the code.
```
Comment (roysagi) [inline Packs/CloudShare/ReleaseNotes/1_0_1.md:4]: I don't think you can edit a pre-existing RN.

File: Packs/CloudShare/Integrations/CloudShare/CloudShare.py (hunk 1 above)
Comment (moishce) [reply Packs/CloudShare/Integrations/CloudShare/CloudShare.py:1183]: also in the py file there is 2 functions (students and student)

File: Packs/CloudShare/ReleaseNotes/1_0_1.md (hunk 3 above)
Comment (ShahafBenYakir) [reply Packs/CloudShare/ReleaseNotes/1_0_1.md:4]: I think this is true

File: Packs/CloudShare/Integrations/CloudShare/CloudShare.py (hunk 1 above)
Comment (ShahafBenYakir) [reply Packs/CloudShare/Integrations/CloudShare/CloudShare.py:1183]: I see, sorry I missed it. So no need to comment. But change the metadata

**Review by ShahafBenYakir — APPROVED** [review]

**Comment by ShahafBenYakir** [comment]
There was a typo in the command name in the release notes - force merging.

**Pull request merged**